src/components/UserInfoTable.tsx

```diff
@@ -0,0 +1,221 @@
+/* eslint-disable */
+/**
+ * Why I disabled eslint: (Ryan)
+ * Linting this file yields two types of errors:
+ * 1. Component definition is missing display name: this comes from the anonymous functions rendering JSX.
+ *    The functions that do this are straight from AntD docs and as far as I'm concerned theyre the only way
+ *    to get search working properly. I'm sure if we had a month to figure out what AntD is doing we could
+ *    remake these functions to make eslint happy but we dont.
+ * 2. Do not use "// @ts-ignore" because it alters compilation errors
+ *    I make my case for why we should use ts-ignore later in the file. tldr: AntD types are icky and hard to work with
+ *    and again, this is idiomatic code straight from the docs. I dont want to spend hours trying to change it to make
+ *    Typescript happy.
+ */
+import { SearchOutlined } from '@ant-design/icons';
+import { Button, Input, Space, Table, Typography } from 'antd';
+import React, { useState } from 'react';
+import Highlighter from 'react-highlight-words';
+import { Link as RouterLink } from 'react-router-dom';
+import { PrivilegeLevel } from '../auth/ducks/types';
+const { Link } = Typography;
+
+export interface UserSummary {
+  firstName: string;
+  lastName: string;
+  email: string;
+  userId: number;
+  privilegeLevel: PrivilegeLevel;
+  phoneNumber: string;
+  profilePicture: string | null;
+  photoRelease: boolean;
+  detailsLink: string;
+}
+const TablePhotoConsent = (text: UserSummary, record: UserSummary) => (
+  <Space size="middle">{record.photoRelease ? 'Yes' : 'No'}</Space>
+);
+const TableActions = (text: UserSummary, record: UserSummary) => (
+  <Space size="middle">
+    <RouterLink to={record.detailsLink}>
+      <Link>View {record.firstName}'s Family Details</Link>
+    </RouterLink>
+  </Space>
+);
+
+interface UserInfoTableProps {
+  users: UserSummary[];
+}
+
+const UserInfoTable: React.FC<UserInfoTableProps> = ({ users }) => {
+  const [searchText, setSearchText] = useState<string>();
```

Comment:
Does this mean it defaults as empty string, or undefined? If we're setting it to `''` in the reset function below, consider making it `''` as default so its technically consistent